Tests/HederaE2ETests/Token/TokenCreate.swift

```diff
@@ -0,0 +1,393 @@
+/*
+ * ‌
+ * Hedera Swift SDK
+ * ​
+ * Copyright (C) 2022 - 2023 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import Hedera
+import XCTest
+
+internal final class TokenCreate: XCTestCase {
+
+    internal func list<T>(amount: Int, element: T) -> [T] {
+        guard amount > 0 else {
+            return []
+        }
+        return Array(repeating: element, count: amount)
+    }
```

Comment:
Given that calling this is `list(amount: x, element: y)` it might be better to just use `Array(repeating: y, count: x)` (especially since this is only called 3 times)

Alternatively,